### PR TITLE
Fix only searching for a single extinction per project

### PR DESCRIPTION
**Pull request by InTheCloudDan**
* Add parameter to `Run` to determine if it should output the flag search(POST or generate CSV). This allows for new manual command to run extinctions only
* fix extinction bug related to count
* update test

### CODE REVIEW

File: coderefs/coderefs.go [hunk 1]
```diff
@@ -68,7 +68,9 @@ func Run(opts options.Options) {
 		CommitTime:       commitTime,
 	}
 
-	handleOutput(opts, matcher, branch, repoParams, ldApi)
+	if output {
+		handleOutput(opts, matcher, branch, repoParams, ldApi)
+	}
```
Comment (jazanne) [inline coderefs/coderefs.go:73]: i understand that this is a function name that already existed, but it was reaaaaaally confusing to me for a few minutes how this enabled us to run just extinctions 😅  - can we rename it? Like, "saveReferences" or something?

**Review by jazanne — APPROVED** [review]
minor comment around function naming. This looks good

File: coderefs/coderefs.go (hunk 1 above)
Comment (InTheCloudDan) [reply coderefs/coderefs.go:73]: @jazanne how about something like `generateHunkOutput` for naming? We should probably split this down more in the future and it's a relatively recent function, for right now though it does manage if it's writing to `csv` or `POST` to API so #namingthingsishard

Comment (jazanne) [reply coderefs/coderefs.go:73]: yea that works for me 👍

**Pull request merged**